mitodl/ocw-www
# Add resource_type facet for resource search and remove content_type filter

Depends on https://github.com/mitodl/open-discussions/pull/3347

#### Pre-Flight checklist


#### What are the relevant tickets?
Closes #68 

#### What's this PR do?
- Adds a "Resource Type" facet for resource search
- Removes the content_type filter that was limiting results to pdf's, videos, and pages.

#### How should this be manually tested?
Checkout https://github.com/mitodl/open-discussions/pull/3347 and 
 - reindex Elasticsearch
 - import some OCW courses if you haven't already.

```
from course_catalog.api import *
prefixes = [
"PROD/6/6.857/Spring_2014/6-857-network-and-computer-security-spring-2014/",
"PROD/MAS/MAS.S62/Spring_2018/mas-s62-cryptocurrency-engineering-and-design-spring-2018/",
"PROD/6/6.01/Fall_2009/6-01-introduction-to-electrical-engineering-and-computer-science-i-fall-2009/",
"PROD/6/6.728/Fall_2006/6-728-applied-quantum-and-statistical-physics-fall-2006/"
]
sync_ocw_courses(course_prefixes=prefixes, blocklist=[], force_overwrite=True, upload_to_s3=False)
```

Start up ocw-www, with `SEARCH_API_URL` pointed to your local open instance.  Go to the search page and switch to resources, you should see a "Resource Types" facet and it should work as expected.  Switch to courses search and you should see the other facets, not "Resource Types".

#### Screenshots

<img width="1157" alt="Screen Shot 2021-03-30 at 9 32 16 AM" src="https://user-images.githubusercontent.com/187676/112997242-efaaa200-913a-11eb-9962-1b7f46cd9591.png">
